upload-api/functions/ucan-invocation-router.js

```diff
@@ -59,10 +59,11 @@ async function ucanInvocationRouter (request) {
 
   const { UPLOAD_API_DID } = process.env;
   const { PRIVATE_KEY } = Config
-  const serviceSigner = getServiceSigner({ UPLOAD_API_DID, PRIVATE_KEY })
+  const servicePrincipal = getServerPrincipal({ UPLOAD_API_DID, PRIVATE_KEY })
```

Comment:
notable here that the return value of `getServerPrincipal` is stored as `servicePrincipal`... suggesting that the function may really be `getServicePrincipal`
```suggestion
  const servicePrincipal = getServerPrincipal({ UPLOAD_API_DID, PRIVATE_KEY })
```

Reply:
ha you're right too. I did start drifting on terminology a bit. good catch :)